packages/techdocs-cli/README.md

```diff
@@ -51,18 +67,34 @@ pip install mkdocs-techdocs-core
 
 Then run `yarn test`.
 
-### Use an example docs project
+#### Running Cypress tests
+
+Running cypress tests requires you to run the CLI locally against our example docs project.
 
-We have created an [example documentation project](https://github.com/backstage/techdocs-container/tree/main/mock-docs) and it's shipped with [techdocs-container](https://github.com/backstage/techdocs-container) repository, for the purpose of local development. But you are free to create your own local test site. All it takes is a `docs/index.md` and `mkdocs.yml` in a directory.
+Start by making sure that you have the example project locally on your computer:
 
 ```sh
 git clone https://github.com/backstage/techdocs-container.git
+```
+
+Run the local version of techdocs-cli against the example docs project:
+
+```sh
+# From the root of this repository run
+# NOTE: This will build the techdocs-cli-embedded-app and copy the output into the cli dist directory
+yarn build --scope @techdocs/cli
```

Comment:
We have a script that builds and runs the CLI locally in one go:
```sh
yarn techdocs-cli serve
```

Reply:
good point, in this case we want to run the cli with another example docs, therefore building separately. But I had added the prepack script to the `cypress:open` script which has now disappeared so will move that back. And with that we don't need this step. Thanks for pointing out! 

Another question - do you think I should move/duplicate the example docs project in here instead? that makes it easier to just run one test command which will do everything we need?

Reply:
Yes, I found myself using the same `mock-docs/index.md` example when I was updating `mkdocs-material`. Maybe we can add the contents of `mock-docs/index.md` to a markdown file inside the Backstage `/docs` folder, on an examples page so we can preview the Backstage `/docs` folder and navigate to this new page in the test. What do you think?

Reply:
Yeah, the /docs documentation is pretty different as its the public docs thats used for (backstage.io) and that microsite uses docusaurus. So my suggestion is to add a docs folder to this package (techdocs-cli) if we want to move it.

Reply:
I moved the docs in this PR, see https://github.com/backstage/backstage/pull/9546/commits/77a13948c15ec3b67b72596fb90a0605a15369a7, which also could make the instructions a bit more clear in the README I think https://github.com/backstage/backstage/pull/9546/commits/a6352f90eaf5b1a1377d39b0d2ed8097dd98d5e2